mkosi/__init__.py

```diff
@@ -1434,6 +1440,12 @@ def parse_ssh_agent(value: str) -> Optional[Path]:
        mkosi --version
 """.format(b=MkosiPrinter.bold, e=MkosiPrinter.reset)
 
+
+def parse_source_target_paths(value: str) -> tuple[Path, Optional[Path]]:
+    split = value.split(':', maxsplit=1)
+    return (Path(split[0]), Path(split[1]) if len(split) > 1 else None)
```

Comment:
Maybe a bit less sigil noise
```suggestion
    src, _, target = value.partition(':')
    return Path(src), Path(target) if target else None
```